examples/real-world/containers/RepoPage.js

```diff
@@ -6,6 +6,7 @@ import Repo from '../components/Repo'
 import User from '../components/User'
 import List from '../components/List'
 import PropTypes from 'prop-types'
+import object from 'lodash/object'
```

Comment:
could we just switch to using `lodash` entry directly in those places? like `import { findKey } from 'lodash'`

Reply:
I'm no expert - I'd have to try it. I'm guessing the syntax would look like:

`import { findKey } from 'lodash/object';`

Reply:
actually `lodash` exports everything from the "main entry" too, u dont have to "scope" it under `object/lodash`

if you dont feel confident in doing this, let me know and I'll merge this as is

Reply:
There we go - that should be sorted.

Cheers